compliance_checker/tests/data/rhgrid.cdl

```diff
@@ -0,0 +1,1048 @@
+netcdf rhgrid {
+dimensions:
+	londim = 128 ;
+	latdim = 64 ;
+	rgrid = 6144 ;
+variables:
+	float PS(rgrid) ;
+		PS:long_name = "surface pressure" ;
+		PS:units = "Pa" ;
+		PS:coordinates = "lon lat" ;
+	float lon(rgrid) ;
+		lon:long_name = "longitude" ;
+		lon:units = "degrees_east" ;
+	float lat(rgrid) ;
+		lat:long_name = "latitude" ;
+		lat:units = "degrees_north" ;
+	float rgrid(rgrid) ;
+		rgrid:compress = "latdim londim" ;
+data:
+
+ PS = _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, 
+    _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, 
```

Comment:
Please don't do full ncdump for this, there's no data in this file, just use ncdump -h.